prow/plugins/assign/assign.go

```diff
@@ -140,6 +162,18 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 	if len(assign) > 0 {
 		log.Printf("Adding assignees to %s/%s#%d: %v", ae.org, ae.repo, ae.number, assign)
 		if err := gc.AssignIssue(ae.org, ae.repo, ae.number, assign); err != nil {
+			nonMembers, emerr := checkMembership(gc, ae.org, assign)
```

Comment:
what will happen if I /assign foo, then I /assign foo again, will I also get the MissingUsers error?